conn.go

```diff
@@ -1552,9 +1552,16 @@ func (cn *conn) processParameterStatus(r *readBuf) {
 		var major1 int
 		var major2 int
 		var minor int
-		_, err = fmt.Sscanf(r.string(), "%d.%d.%d", &major1, &major2, &minor)
-		if err == nil {
-			cn.parameterStatus.serverVersion = major1*10000 + major2*100 + minor
+		// if the version string contains at least major1, set the server version
+		n, _ := fmt.Sscanf(r.string(), "%d.%d.%d", &major1, &major2, &minor)
```

Comment:
Hi @xstevens ! 

I think it still makes sense to check for error here because **Sscanf** function can return an error and n > 0 at the same time. What do you think about?